openmdao/recorders/web_recorder.py

```diff
@@ -456,3 +406,9 @@ def convert_to_list(self, obj):
             return []
         else:
             return obj
+
+    def close(self):
+        """
+        Close.
+        """
+        pass
```

Comment:
This should be addressed in the most recent commit, where I took `close()` out of the server recorder altogether and simply put in `pass` as the base implementation of `close`. Thus, it can be implemented by the subclass if needed, but isn't required.